go/host/rpc/enclaverpc/enclave_client.go

```diff
@@ -227,8 +230,14 @@ func (c *Client) ExecuteOffChainTransaction(encryptedParams common.EncryptedPara
 	if err != nil {
 		return nil, err
 	}
-	if response.Error != "" {
-		return nil, errors.New(response.Error)
+	if len(response.Error) > 0 {
+		// todo - handle other types of errors
```

Comment:
I think it would be prudent to handle this todo in this PR, and return an `errors.New(response.Error)` error in the case it's not a SerialisableError.

Reply:
this todo was old